### PR TITLE
rtl_443 20.11 (new formula)

### PR DESCRIPTION
**Pull request by drfruct**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/rtl_433.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
+class Rtl433 < Formula
+  desc "Program to decode radio transmissions from devices"
+  homepage "https://github.com/merbanan/rtl_433"
+  url "https://github.com/merbanan/rtl_433/",
+    using:    :git,
```
Comment (SMillerDev) [inline Formula/rtl_433.rb:5]:
```suggestion
  url "https://github.com/merbanan/rtl_433.git",
```

**Review by SMillerDev — COMMENTED** [review]
Is there no tarbal of the release? Otherwise we'll want to use that one.

File: Formula/rtl_433.rb (hunk 1 above)
Comment (drfruct) [reply Formula/rtl_433.rb:5]: The program version is formed based on the last tag of the git repository.
If I use tarball, the output of the program version will return null.

This code is a formatted version of the program:
https://github.com/merbanan/rtl_433/blob/master/cmake/Modules/GetGitRevisionDescription.cmake

File: Formula/rtl_433.rb [hunk 2]
```diff
@@ -0,0 +1,41 @@
+class Rtl433 < Formula
+  desc "Program to decode radio transmissions from devices"
+  homepage "https://github.com/merbanan/rtl_433"
+  url "https://github.com/merbanan/rtl_433.git",
+    tag:      "20.11",
+    revision: "0bb4672b1afb716d107b3060afc2d80a2159dbef"
```
Comment (carlocab) [inline Formula/rtl_433.rb:6]:
```suggestion
      tag:      "20.11",
      revision: "0bb4672b1afb716d107b3060afc2d80a2159dbef"
```
Let's follow the way other formulae align the `url` arguments.

File: Formula/rtl_433.rb [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+      system "cmake", "..", *std_cmake_args
+      system "make", "install"
+    end
+  end
+
+  test do
+    resource("test_cu8").stage testpath
+    resource("expected_json").stage testpath
+
+    expected_output = shell_output("cat #{testpath}/g001_433.92M_250k.json")
```
Comment (carlocab) [inline Formula/rtl_433.rb:36]:
```suggestion
    expected_output = (testpath/"g001_433.92M_250k.json").read
```
I don't think you need `cat` here.

**Review by carlocab — DISMISSED** [review]

**Review by SMillerDev — APPROVED** [review]
Thanks @drfruct  ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Pull request closed without merging**